tests/provision_test.py

```diff
@@ -0,0 +1,75 @@
+ # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+import unittest
+import subprocess
+from shlex import split
+import json
+
+from google.cloud.container_v1.services import cluster_manager
+
+class TestGKECluster(unittest.TestCase):
```

Comment:
what will invoke this test?

Reply:
"python3 provision_test.py" will invoke the tests.

Reply:
you should document it in readme.md.
Also please create a "provisioning" folder under "tests" and move your tests there